FogCreek/Glitch-Community
# Sudden kick: componentize much of `/includes`

## Links
* Remix link: https://sudden-kick.glitch.me

## GIF/Screenshots:
![Screen Shot 2019-06-17 at 1 44 48 PM](https://user-images.githubusercontent.com/938722/59627597-9cee2a80-910c-11e9-8492-81294031b77b.png)
![Screen Shot 2019-06-17 at 2 31 05 PM](https://user-images.githubusercontent.com/938722/59627598-9cee2a80-910c-11e9-9f27-ef60c4063009.png)

## Changes:
* move ErrorBoundary, AuthDescription, ProjectActions, Uploader to new directories (no changes)
* componentize CollectionAvatar
* inline NameConflict into teams page

## How To Test:
* verify CollectionAvatar highlight color coordinates with the collection's cover color
* i have no idea how to test NameConflict without editing the code to make it visible
